swift/apple/FirezoneKit/Sources/FirezoneKit/Views/WelcomeView.swift

```diff
@@ -44,19 +41,8 @@ struct WelcomeView: View {
         """).multilineTextAlignment(.center)
           .padding(.bottom, 10)
         Button("Sign in") {
-          if !tapped {
-            tapped = true
-
-            DispatchQueue.main.async {
-              model.signInButtonTapped()
-            }
-
-            DispatchQueue.main.asyncAfter(deadline: .now() + 0.2) {
```

Comment:
Turns out all of this debounce logic was required due to my previously buggy use of DispatchQueue 🙃 . Works great now.